lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/include/Config/PlatformConfig.hpp

```diff
@@ -0,0 +1,303 @@
+#ifndef PLATFORM_CONFIG_HPP
+#define PLATFORM_CONFIG_HPP
+
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#pragma once
+
+#include <unordered_map>
+#include "config_legacy.h"
+#include "EnumMappings.h"
+#include "eos_init.h"
+#include "eos_auth_types.h"
+#include "eos_ui_types.h"
+#include "json_helpers.h"
+#include "ProductionEnvironments.hpp"
+#include "Config/ClientCredentials.hpp"
+#include "Config/Config.hpp"
+
+namespace pew::eos::config
+{
+    /**
+     * \brief Describes the configuration options for a platform that wants to
+     * interact with the EOS SDK.
+     */
+    struct PlatformConfig : Config
+    {
+    public:
+        /**
+         * \brief Used to statically store the cache directory once it has been determined.
+         */
+        static inline std::string s_cache_directory;
```

Comment:
Why are these static?

Reply:
This issue is the same as a comment you made in another conversation - so I am marking this one as resolved and we will discuss it there.